var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -48,6 +48,9 @@ class Mfem(Package, CudaPackage, ROCmPackage):
     # other version.
     version("develop", branch="master")
 
+    # FIXME: replace branch with tarball when v4.5.2 is released
+    version("4.5.2", branch="mfem-4.5.2-dev")
```

Comment:
`4.5.2` alpha, beta, or release candidate is a better designation for work in progress. You can then deprecate `4.5.2a` when `4.5.2` is released. See version comparisons: https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison